docs/cassandra.adoc

```diff
@@ -19,7 +19,7 @@ If you are using Cassandra 2.2 or higher you need to explicitly enable thrift so
 Do so by running `./bin/nodetool enablethrift`.
 [NOTE]
 If security is enabled on Cassandra, the user must have `CREATE permission on <all keyspaces>`,
-otherwise the keyspace must be created ahead of time by an administrator.
+otherwise the keyspace must be created ahead of time by an administrator including the required tables or the user must have `CREATE permission on <the configured keyspace>`.
```

Comment:
We should create a cql file with the required tables in it to make it easier for the Cassandra administrator to create the tables on behalf of the JanusGraph user. Then add to these docs on where to find the cql file in the JanusGraph distribution and how to load it with cqlsh.

Reply:
[janusgraphCassandraTables.txt](https://github.com/JanusGraph/janusgraph/files/2122228/janusgraphCassandraTables.txt)

I would say this is the current situation. The required tables are not defined in the cql module but in core (Backend.java). Placing the file in core seems to be wrong and placing it in cql module it might be hard to keep this document updated.

I started to use janusgraph not a long time ago. As a developer I set it up locally and janusgraph created the keyspace containing all the tables. This way I had the current schema. Bad would just be when janusgraph would create some tables later during runtime.

If a cassandra admin does not want janusgraph to do any schema modification, then I think the knowhow how to handle a cql file should already be there.

Reply:
I suggest placing the file under `janusgraph-dist/src/assembly/static/conf/cassandra/`, which would end up in the zip distribution under `conf/cassandra/`. I wouldn't expect the file to be updated frequently.

Would you be able to add the file and a note in the docs pointing to the file location? Thanks.

Reply:
Thank you for your input. I added the file and updated the documentation.